src/hal/hal.cpp

```diff
@@ -42,7 +42,7 @@ static void hal_io_init () {
     hal_interrupt_init();
 }
 
-// val == 1  => tx 1
+// val == LMIC_ANTENNA_SWITCH_TX  => tx
 void hal_pin_rxtx (u1_t val) {
     if (plmic_pins->rxtx != LMIC_UNUSED_PIN)
         digitalWrite(plmic_pins->rxtx, val);
```

Comment:
With a wiring change, we'd say:
```c
digitalWrite(plmic_pins->rxtx, val != plmic_pins->rxtx_rx_active);
```
The `rxtx_rx_active` field would default to zero according to C defaults, so no need to change source code on existing deployments, as long as `rxtx_rx_active` is after all the existing fields in the `lmic_pins` structure.